src/index.test.js

```diff
@@ -0,0 +1,93 @@
+const capitalize = require('./index');
+const reverseString = require('./reverse');
+const calculator = require('./calculator');
+const caesar = require('./caesar');
+const analyzeArray = require('./array');
+
+test('Capitalize first string passes ', () => {
+  expect(capitalize('string')).toBe('String');
+});
+test('Capitalize first string fails', () => {
+  expect(capitalize('string')).not.toMatch(/string/);
+});
+
+test('Reverse string pass', () => {
+  expect(reverseString('string')).toMatch(/gnirts/);
+});
```

Comment:
If you match against a regex, it is going to pass even if it is just a substring. So if for some reason your `reverseString` function modifies the output, it could also pass.

For example, if the result of `reverseString('string')` is `'abc gnirts efg'`, the test is going to pass:

```js
expect("abc gnirts efg").toMatch(/gnirts/); // PASS
```

Please use `toBe('string')` to match against the expected string.

Applies for all cases of `toMatch` unless you are really [testing against regex a pattern](https://jestjs.io/docs/en/using-matchers#strings).

Reply:
OK, perfect. 
According to: Your caesar cipher doesn't take into account uppercase letters.
The website recommended, to read about it, has the Caeser cipher case insensitive, That's why I did it like that.

Reply:
100% percent agree with you. That's why it is under OPTIONAL. I just thought this was way easy for you and wanted to through something more challenging to improve/enhance your code. But only if you are up to it. No stress about it.